src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -272,7 +272,7 @@ def update_address_ids(
     units_that_move_ids: pd.Index,
     starting_address_id: int,
     address_id_col_name: str,
-) -> Tuple[pd.DataFrame, pd.DataFrame, int]:
+) -> Tuple[pd.DataFrame, int]:
```

Comment:
Update docstring?

Reply:
I did! Sorry, I might not have done this until after you started your review 😅